crates/query-engine/translation/src/translation/mutation/v1/delete.rs

```diff
@@ -157,7 +160,8 @@ mod tests {
         let mut arguments = BTreeMap::new();
         arguments.insert("user_id".to_string(), serde_json::Value::Number(100.into()));
 
-        let result = translate_delete(&mut state, &delete, &arguments).unwrap();
+        let result =
+            Env::with_empty(|env| translate_delete(&env, &mut state, &delete, &arguments).unwrap());
```

Comment:
This really cannot be `let env = Env::new(); translate_delete(...);`?

Reply:
Well, it could, if you prefer giving all the arguments that `new(..)` takes, including having a separate `Metadata` value that `new(..)` can borrow. (at least I know rustc _sometimes_ asks me to make a separate binding when I pass references to newly constructed values).

I considered this for a bit, but in the end I came up with `with_empty(..)` so the test wouldn't have to concern itself so much with what makes up an Env.

Feel free to make any changes to this to match your preferences :-)